primitives/src/migrate.rs

```diff
@@ -174,3 +175,18 @@ pub fn kill_item(module: &[u8], item: &[u8]) {
     prefix[16..32].copy_from_slice(&Twox128::hash(item));
     kill_prefix(&prefix)
 }
+
+/// Copies a single or double map storage item under a new module prefix and removes the map from
+/// the old module prefix.
+///
+/// Migrations mapping to `None` are silently dropped from storage.
+pub fn copy_map_rename_module<T: Decode + Encode>(
```

Comment:
Better to call this `move` instead of `copy` since the `drain` function being sued below will delete the old entries.